src/driver/packages.ml

```diff
@@ -1,5 +1,7 @@
 (* Packages *)
 
+type pkgdir = string * Fpath.t
```

Comment:
I'm not keen on the name `pkgdir` to refer to something that isn't just a `dir`

Reply:
In fact, I'm not keen on pairing the two things together. They're used for quite different purposes - the name is only there because we need the package name for the 'current-package' argument, while the directory is used in many other places. I'll try to separate them back out again.